projects/ngx-formentry/src/form-entry/form-renderer/form-renderer.component.scss

```diff
@@ -23,15 +23,15 @@
 
 .nav-link-text {
   margin-top: 0.125rem;
-  white-space: nowrap;
```

Comment:
I think it's important to keep this around for visual consistency. Having the page name truncated when the titles are too long isn't necessarily a bad trade-off when the full page names are listed on the left panel imo.

Reply:
Agreed. Thanks.